### ISSUE TITLE
any update about ios module ?

### ISSUE THREAD

**Issue by HusseinElGanzory** [A-Hussamadin/HyperpayReactNativeDemo#3]

**Comment by anastely** [comment]
Have you figured out it?

**Comment by HusseinElGanzory** [comment]
> Have you figured out it?

Yes, al humdallah

**Comment by anastely** [comment]
> > Have you figured out it?
> 
> Yes, al humdallah 

Good, can u help me in this case, I'm stuck :(? 
How can i chat with u if u don't mind:)

**Comment by HusseinElGanzory** [comment]
Yes my bro where are you from ? We can chat on messenger or WhatsApp

**Comment by anastely** [comment]
Palestine:) here's my number, 
+972597093830
Please let me know when you see this !

**Comment by anastely** [comment]
> Yes my bro where are you from ? We can chat on messenger or WhatsApp 

Palestine:) here's my number, 
+972597093830
Please let me know when you see this !

**Comment by anastely** [comment]
@HusseinElGanzory Are you saw the comment?

**Comment by HusseinElGanzory** [comment]
Hi my bro can you send messages on messenger u can search with hussein ElGanzory

**Comment by anastely** [comment]
this [your account?](https://www.facebook.com/ElGanzoryHussein)
if yes check inbox :)

**Comment by HammzaHM** [comment]
@HusseinElGanzory  Please can you help me with the IOS Module ??? I AM stuck please help me

**Comment by HammzaHM** [comment]
@anastely If you got the IOS Module Code please send it to me via this email: hmemm.hamza1@gmail.com
